app/src/main/java/com/github/se/wanderpals/ui/screens/dashboard/DashboardSuggestion.kt

```diff
@@ -0,0 +1,86 @@
+package com.github.se.wanderpals.ui.screens.dashboard
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.wanderpals.model.data.GeoCords
+import com.github.se.wanderpals.model.data.Stop
+import com.github.se.wanderpals.model.data.Suggestion
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+
+@Composable
+fun DashboardSuggestion(suggestion: Suggestion) {
+  Card(modifier = Modifier.padding(8.dp).testTag("suggestion${(suggestion.suggestionId)}")) {
+    Column(
+        modifier =
+            Modifier.border(1.dp, Color.Black, shape = RoundedCornerShape(16.dp)).padding(16.dp)) {
+          Row(
+              modifier = Modifier.fillMaxWidth(),
+              horizontalArrangement = Arrangement.SpaceBetween) {
+                Text(text = suggestion.stop.title, modifier = Modifier.testTag("stopTitle"))
```

Comment:
You can limit the lines to only receive a certain number of characters `MAX_ONE_LINE_LENGTH` for `suggestion.stop.title`. If it reaches this number, it will go to the next line to display the remaining `suggestion.stop.title`.

See the first point of the overall comment. This will resolve the issue mentioned in it.

Reply:
Fixed